packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -44,6 +45,10 @@ describe('masternode', () => {
     await container.call('createtoken', [metadata])
     await container.generate(1)
 
+    const payload: { [key: string]: string } = {}
+    payload[address] = '100@0'
+    await container.call('utxostoaccount', [payload])
```

Comment:
I think you can write like this for brevity
```suggestion
    await container.call('utxostoaccount', [{
      [address]: '100@0'
    }])
```